polkadot/xcm/src/v2/junction.rs

```diff
@@ -41,7 +41,7 @@ pub enum Junction {
 	/// An 8-byte index for an account of a specific network that is respected as a sovereign
 	/// endpoint within the context.
 	///
-	/// May be used when the context is a Frame-based chain and includes e.g. an indices pallet.
+	/// May be used when the context is a Frame-based chain and includes e.g. an indices' pallet.
```

Comment:
```suggestion
	/// May be used when the context is a Frame-based chain and includes e.g. an indices pallet.
```

It's referring to "an indices pallet", not the pallet of the indices.